python/tvm/topi/mali/conv2d.py

```diff
@@ -545,6 +577,19 @@ def _alter_conv2d_layout(attrs, inputs, tinfos, out_type):
         return None
 
 
+@conv2d_winograd_nhwc.register(["mali"])
+def conv2d_winograd_nhwc_cuda(
```

Comment:
Should this be called `conv2d_winograd_nhwc_mali`?

Reply:
Yes. Thanks! New commit will resolve this!